DP03_05_Using_User-Supplied_Catalogs.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<p>this cell had the following flake8 errors:</p>

<pre class="ql-syntax" spellcheck="false">pycodestyle INFO: 4:63: W291 trailing whitespace
pycodestyle INFO: 6:28: W291 trailing whitespace
pycodestyle INFO: 7:80: E501 line too long (129 &gt; 79 characters)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>